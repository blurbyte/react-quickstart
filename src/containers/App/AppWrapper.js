import styled from 'styled-components';

const AppWrapper = styled.div`
  min-width: 100%;
  min-height: 100%;
`;

export default AppWrapper;
