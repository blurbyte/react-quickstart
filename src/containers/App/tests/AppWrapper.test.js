import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import AppWrapper from '../AppWrapper';

describe('<AppWrapper />', () => {
  it('should render correctly', () => {
    // component init
    const component = renderer.create(
      <AppWrapper />
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
