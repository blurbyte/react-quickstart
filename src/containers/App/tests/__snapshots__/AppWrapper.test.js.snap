// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppWrapper /> should render correctly 1`] = `
.cXbYyw {
  min-width: 100%;
  min-height: 100%;
}

<div
  className="cXbYyw"
/>
`;
